### DISCUSSION TITLE
无法启动ALIST，管理页面无法访问

### DISCUSSION THREAD

**Discussion by simon333888** [AlistGo/alist#2009] [Invalid question]
在CMD中运行.\alist.exe server命令后返回这个图上的结果
![截屏_2022-10-14_13-38-18](https://user-images.githubusercontent.com/40132809/195770475-75298869-9e20-4428-921a-7ff2815d0935.png)
然后浏览器地质输入127.0.0.1:5244后无法访问![image](https://user-images.githubusercontent.com/40132809/195770985-c57de3d1-6432-4c79-98ef-183867aab117.png)
也不知道哪里出了问题，之气碰运气登陆上过两次，一头雾水。

**Comment by simon333888** [comment]
有没有老铁遇到过，帮忙解答一下

**Reply by 12189108** [reply]
打开“程序与功能”
点击“启动或关闭Windows功能”
选中“Internet Information Services”

**Reply by simon333888** [reply]
多谢指导，已解决

**Reply by crazypeng2828** [reply]
我也遇到这个问题，开了Internet Information Services也解决不了。

**Comment by xiaoenfeng** [comment]
我也是 一直显示：无法访问此网站127.0.0.1 拒绝了我们的连接请求。
上面办法也试了，还是不行o(╥﹏╥)o

**Comment by q965502450** [comment]
重新安装-再更新一遍-启动

**Comment by LeaiFish** [comment]
我是在istore里的，重装了也还是不行😭

**Comment by imwcc** [comment]
我发现安装宝塔会开启防火墙，关闭解决

**Comment by dongfanghao168** [comment]
我也是进不去管理页面

**Comment by snai9** [comment]
我也遇到了，不知道这个问题最终要怎么解决？而且我第一次是可以进的，第二天就不能进了

**Reply by nanoxxs** [reply]
命令窗关闭后 服务就终止了 当然无法访问  
解决方法：创建一个VBS脚本 设置为开机自启
脚本内容：

Set ws = CreateObject("Wscript.Shell")
ws.run "C:\alist-windows-amd64\alist.exe server",vbhide

把C:\alist-windows-amd64改为你自己在电脑上存放该文件的目录

具体操作为
1、创建一个文本文件，文件后缀改为.VBS；
2、复制以上脚本，保存后拖到C:\alist-windows-amd64目录下；
3、在该目录下右键此脚本，创建快捷方式；
4、win+r，输入shell:startup，调出开机自启目录；
5、将脚本的快捷方式拖到开机自启目录中（注意一定是快捷方式，不是原脚本文件）

这样在每次开机后便会自动开启alist服务
